www/docs/en/dev/guide/appdev/security/index.md

```diff
@@ -103,5 +103,4 @@ When running Cordova on Android, using `android:debuggable="true"` in the applic
 ## Recommended Articles and Other Resources
 
 * [HTML5 Security cheat sheet, detailing how to secure your HTML5 application](https://www.owasp.org/index.php/HTML5_Security_Cheat_Sheet)
-* [Phonegap's article on device security, such as using encrypted data](https://github.com/phonegap/phonegap/wiki/Platform-Security)
```

Comment:
Link redirects to the "Home" screen of github phonegap. The original article doesn't appear to exists.